test/test_quantized.py

```diff
@@ -95,15 +98,24 @@ def test_qrelu(self, X):
                                    dtype=torch_type)
 
         ops_under_test = {
-            'ops.quantized': torch.ops.quantized.relu,
             'native': torch.relu,
-            'nn.functional': torch.nn.functional.relu
+            'nn.functional': torch.nn.functional.relu,
         }
 
         for name, op in ops_under_test.items():
             qY_hat = op(qX)
             self.assertEqual(qY, qY_hat, message="{} relu failed".format(name))
 
+        ops_under_test_inline = {
+            'inline native': torch.relu_,
+            'inline nn.functional': torch.nn.functional.relu_,
+        }
+
+        for name, op in ops_under_test_inline.items():
+            qY_hat = qX.clone()
+            op(qY_hat)
```

Comment:
Can you name `op` to `op_` so that someone scanning the code will be immediately aware this is in-place?